### ISSUE TITLE
Menus: Create placeholder localization service

### ISSUE THREAD

**Issue by lyonsil** [paranext/paranext-core#418]
As a Platform.Bible user, I want menus to be able to be displayed in whatever language I configure in the UI.

----

Until #331 is designed and rolled out, we need a simple way for menus to be able to specify and reference localization keys.  The purpose of this ticket is to build a very simple key/value lookup service with an API for localization purposes.  This placeholder localization service will be used in the construction of JSON documents that fully describe a menu that the renderer should draw.  Once the real localization service is in place, this placeholder service can be discarded and the menu service updated to use the real one.

The overall [menu contribution design document](https://docs.google.com/document/d/1N9BVj5w1b3H90Sjjc1lRN2zbalL-beDHmh4jxxExtiw/edit?usp=drive_link) has many more details, but most of that is not needed for this ticket.  What we want is a service where something can ask "What is the localized text to display for `%some_localization_key%` given the user's desired UI language?".  Since we don't have a way for the "desired UI language" to be specified right now, this can be simplified as "What is the localized text to display for `%some_localization_key%`?"

If anyone rates this a large or XL size ticket, we should really talk about why.
